src/keyboard.rs

```diff
@@ -176,10 +176,11 @@ pub enum NativeKeyCode {
     Windows(u16),
     MacOS(u32),
     XKB(u32),
+    Web(&'static str),
```

Comment:
I think this can also be fixed by using the [`#[serde(deserialize_with = "func")]`](https://serde.rs/field-attrs.html#deserialize_with) attribute, but a `String` would also be fine, as this should only happen very rarely, and I don't expect that people would want to match on this value under normal circumstances.

Reply:
Actually, using `String` would make `NativeKeyCode` no longer `Copy`.

Reply:
I've just removed the contents of the `Web` variant for now.

Reply:
After thinking about it a bit, I still think that it should be a `String`. Loosing the `Copy` trait is a bit unfortunate but it still seems to be the most reasonable thing to do, especially because as you said, this type should be rarely used.